xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -645,9 +638,6 @@ void CDRMUtils::DestroyDrm()
   RestoreOriginalMode();
 
   drmDropMaster(m_fd);
-  close(m_fd);
```

Comment:
imo if you have a function called `DestroyDrm` you should also close the fd here